web/shapeworks/src/components/RenderControls.vue

```diff
@@ -313,6 +338,15 @@ export default {
                 v-model="showDifferenceFromMeanMode"
                 label="Show difference from mean"
             />
+            <v-text-field
+                v-if="showSamplePageSelector"
+                :value="deepSSMSamplePage"
+                label="Sample Page"
+                min="0"
+                step="1"
+                @input="onChangeSamplePage"
+                type="number"
+            />
```

Comment:
Is it possible to use built-in pagination from the `v-data-table` rather than an extra input?

Reply:
This would require some decent amount of change. The data table would need to pull only generated samples or original, add hooks, etc. right now the data is just coming from a CSV file.

Reply:
I think this would fit into future work